app/views/overview.html

```diff
@@ -6,16 +6,15 @@
       <!-- Getting started -->
       <div class="empty-state-message text-center">
         <div ng-if="project.metadata.name | canIAddToProject">
-          <h2>Get started with your project.</h2>
-          <p class="gutter-top">
-            Use your source or an example repository to build an application
-            image, or add components like databases and message queues.
+          <h2 translate>Get started with your project.</h2>
+          <p class="gutter-top" translate>
+            Use your source or an example repository to build an application image, or add components like databases and message queues.
```

Comment:
I'm curious if there is an automated test for validating we don't break any translation strings. I could see us accidentally changing a word in a sentence like this & finding that the translation fails.

Reply:
long term the assumption has to be that mid-release there is no guarantee that the translations are correct. Then every 3.x release we would have a translation phase. But we cant even get to that point until we have everything going through gettext, and then we have to figure out where in the release process the translation phase would live, because it means adding a "string lockdown" date

Reply:
@jwforres yes, we are came from china, we have always deploy many more instance for users, but user prefer native language interface, so we need quick build the translation process workflow,  let the translation cycle is working on process. thanks a lot.